### PR TITLE
save yaml file under DDLN_CSV_BB

### PR DESCRIPTION
**Pull request by chichinp**
Added ddln.yaml file while dumping annotation to DDLN_CSV_BB ZIP file.
It contains different hard-coded values (for now), parsed phabricator task from CVAT Task name, assets hash from ddln_id file and date of export.

Phabricator: https://p.daedalean.ai/T6082

### CODE REVIEW

File: cvat/apps/annotation/ddln_spotter.py [hunk 1]
```diff
@@ -20,6 +20,23 @@
     ],
 }
 
+YML_TEMPLATE = """sources:
+  - {ddln_id}
+date: {curr_date}
+team: {team}
```
Comment (gyakushev) [inline cvat/apps/annotation/ddln_spotter.py]: this is not valid YAML
the task is filed incorrectly, fix the format to produce valid YAML

**Review by gyakushev — CHANGES_REQUESTED** [review]

File: cvat/apps/annotation/ddln_spotter.py (hunk 1 above)
Comment (chichinp) [reply cvat/apps/annotation/ddln_spotter.py]: fixed!

File: cvat/apps/annotation/ddln_spotter.py [hunk 2]
```diff
@@ -20,6 +20,23 @@
...
+team:
+  - group: {group}
+  - mapping: {map_file}
+phabricator: {task}
+tool:
+  - name: CVAT
+    version: 2.3
+  - name: merge tool https://git-ng.daedalean.ai/daedalean/exp-devtools/src/branch/master/annotations/multi/process_annotations_msq.py
+    version: pre-commit
+invalid:
```
Comment (gyakushev) [inline cvat/apps/annotation/ddln_spotter.py:35]: remove empty fields

File: cvat/apps/annotation/ddln_spotter.py [hunk 3]
```diff
@@ -20,6 +20,23 @@
...
+  - {ddln_id}
+date: {curr_date}
+team:
+  - group: {group}
+  - mapping: {map_file}
+phabricator: {task}
+tool:
+  - name: CVAT
+    version: 2.3
+  - name: merge tool https://git-ng.daedalean.ai/daedalean/exp-devtools/src/branch/master/annotations/multi/process_annotations_msq.py
```
Comment (gyakushev) [inline cvat/apps/annotation/ddln_spotter.py:33]: separate name from command line.
```
name: multi/process_annotations_msq # or similar
cmd: merge tool https://git-ng.daedalean.ai/daedalean/exp-devtools/src/branch/master/annotations/multi/process_annotations_msq.py
```

the name should be unique identifier that is easy to group by

File: cvat/apps/annotation/ddln_spotter.py [hunk 4]
```diff
@@ -20,6 +20,23 @@
...
+    version: pre-commit
```
Comment (gyakushev) [inline cvat/apps/annotation/ddln_spotter.py:34]: `pre-commit` is not a useful version, it doesn't tell me anything when I look at this file next year

File: cvat/apps/annotation/ddln_spotter.py [hunk 5]
```diff
@@ -51,11 +68,43 @@ def dump(file_object, annotations):
...
+    map_file = "task_mapping.csv"
+    id_file_path = None
+
+    task = annotations.meta['task']['name']
+    task = task.split('_')[0]
+    curr_date = annotations.meta['dumped'][:10]
+
+    dir_prefix = "/home/django/share/incoming/{}/"
+    for dirp, dirn, files in os.walk(dir_prefix.format(task)):
+        if "spo_" in dirp:
```
Comment (alexanderGerbik) [inline cvat/apps/annotation/ddln_spotter.py:82]: Is it possible that task directory might have multiple 'spo_' directories and should we fail in this case?
I think this case could be handled this way:
```python
def get_annotation_request_id(task_name):
	import pathlib
	base_dir = pathlib.Path("/home/django/share/incoming")
	task_dir = base_dir.joinpath(task_name)
	ddln_id_files = list(task_dir.glob("spo_*/ddln_id"))
	if len(ddln_id_files) != 1:
		return None
	return ddln_id_files[0].read_text().strip()
```

**Review by gyakushev — APPROVED** [review]
I see no blockers. Thanks!

**Pull request merged**